+jrclust/+views/plotFigTime.m

```diff
@@ -34,24 +57,47 @@
         fgTimes2 = [];
         figTitle = sprintf('Unit %d (black); (press [H] for help)', selected(1));
     end
+   
+    binlimits = [min(bgFeatures) max(bgFeatures)]+eps;     
+    
+    % remove foreground events from background cluster    
+    bg_idx = ~ismember(bgTimes,union(fgTimes,fgTimes2));
```

Comment:
Good idea. Might be better in the long run to supply `getFigTimeFeatures` which a parameter of units to skip, but this is fine for now.

Reply:
Yes that would be more general. Is there a use case for that you can imagine, other than excluding the selected clusters from the background?

Reply:
I was thinking just so you could get the full 10000 background spikes, but in most cases a random sample would get you pretty close to that. Otherwise, I can't think of a good use case for that, so we can table that idea for now.